inst/extdata/aglu/FAO/FAO_USA_an_Prod_t_PRODSTAT.csv

```diff
@@ -1,6 +1,8 @@
 # File: FAO_USA_an_Prod_t_PRODSTAT.csv
 # Title: FAO USA historical production animal products
 # Units: tonnes
+# Column types: ciciciiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii 
```

Comment:
Are we sure we want these (and related values) to be parsed as integers? I feel like production values, in tons, should allow fractions (i.e. should be parsed as "n").

Since I'm guessing we have fewer places where we expect integers than "numbers" (doubles), perhaps it would be safer to re-generate these files with the newer version of `readr` so all of these parse as numbers and then manually go back and fill in the few places where we actually expect integers (e.g. years).

Reply:
I think @pkyle said that, for whatever reason, these are all "integers"--that's how they're reported in the file from the FAO; this isn't a `readr`-versioning issue. We could still force them to be read as numeric, of course, but that breaks the existing code.  😢

Reply:
Fair enough. This just caught my eye and I just wanted to make sure it was intentional.

Reply:
Wait, it's the opposite--these should be doubles, not integers. Sorry if I screwed something up here--any numeric value (whether in tonnes or ktoe) should be a double. It's a common practice in these datasets to use a small unit and then not write out any decimals.

Reply:
Sorry. I mean, they're reported as "integers", i.e. with no fractional part, and read in as such, but then quickly become numerics as they're computed on in [module_aglu_LB132.ag_an_For_Prices_USA_C_2005](https://github.com/JGCRI/gcamdata/blob/master/R/zchunk_LB132.ag_an_For_Prices_USA_C_2005.R). This is what currently happens, anyway, and matches the behavior in the old data system.